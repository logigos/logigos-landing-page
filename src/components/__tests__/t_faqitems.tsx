import React from 'react'
import renderer from 'react-test-renderer'

import FaqItem from '../common/FaqsItem'

describe('FAQItem', () => {
    const Mockfaqs = {
        Q: 'Question',
        A: 'Anwser',
    }

    const handleOnClick = () => console.log('ok')

    it('(NOT EXPAND) renders correctly', () => {
        const tree = renderer
            .create(
                <FaqItem
                    expand={false}
                    handleOnClick={() => handleOnClick()}
                    {...Mockfaqs}
                />
            )
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
    it('(EXPAND) renders correctly', () => {
        const tree = renderer
            .create(
                <FaqItem
                    expand={true}
                    handleOnClick={() => handleOnClick()}
                    {...Mockfaqs}
                />
            )
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})
