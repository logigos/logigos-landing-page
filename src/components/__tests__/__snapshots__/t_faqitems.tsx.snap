// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAQItem (EXPAND) renders correctly 1`] = `
<div
  className="sc-AxhUy eKGhvn"
  onClick={[Function]}
>
  <test-file-stub
    className="sc-AxjAm jnIgPP"
  />
  <p
    className="sc-AxiKw lfFHDL"
  >
    Question
  </p>
  <p
    className="sc-AxhCb gBEShw"
  >
    Anwser
  </p>
</div>
`;

exports[`FAQItem (NOT EXPAND) renders correctly 1`] = `
<div
  className="sc-AxhUy koLheE"
  onClick={[Function]}
>
  <test-file-stub
    className="sc-AxirZ urQxH"
  />
  <p
    className="sc-AxiKw exFDCh"
  >
    Question
  </p>
  <p
    className="sc-AxhCb elSiqH"
  >
    Anwser
  </p>
</div>
`;
